README.md

```diff
@@ -81,6 +84,26 @@ to get document access.
 - [bit.ly/kueue-controller-design](https://bit.ly/kueue-controller-design)
 presents the detailed design of the controller.
 
+## Roadmap
+
+This is a high-level overview of the main priorities for 2023, in expected order of release:
+
+- Job preemption to reclaim borrowed quota and to accommodate high priority jobs [#83](https://github.com/kubernetes-sigs/kueue/issues/83), this is planned for v0.3
+- More advanced allocation strategies for better resource sharing [#312](https://github.com/kubernetes-sigs/kueue/issues/312)
+- Integration with cluster-autoscaler for guaranteed resource provisioning
+- Integration with common custom workloads [#74](https://github.com/kubernetes-sigs/kueue/issues/74):
+  - Kubeflow (TFJob, MPIJob, etc.)
+  - Spark
+  - Workflows (Tekton, Argo, etc.)
+
+### Long-Term Features
+
+These are features that we aim to have in the long-term, in no particular order:
+
+- Multi-cluster support
+- Dashboard for management and monitoring for administrators
+- Budget support
```

Comment:
maybe on these two it's more realistic to make progress in 2023, as opposed to multi-cluster

Reply:
I don't think the roadmap needs to contain every possible feature. Just those that are bigger effort.

Reply:
Maybe you can add #28 behind budget support, then if someone want to learn details or background, they can read or ask in that issue.

Reply:
updated, actually there's no particular order, just a rough plan.